english/covid-19/2/2.md

```diff
@@ -0,0 +1,7 @@
+You stayed inside the room and shut the door close. After your eyes adjusted to the darkness, you checked if there is something in the room that's useful. 
+
+Surprisingly, you found a really cool sword and took it.
+
```

Comment:
> At least make one progressive link here which leads to somewhere else other than (../WIP.md)

same here (commenting just for the sake of marking these files 😅)

Reply:
Thanks for your comments. But, I don't know how to progress the story. That's why I opened this PR. So, let any others complete it 😉